### DISCUSSION TITLE
[Laravel recipe] Skip task if .env is missing

### DISCUSSION THREAD

**Discussion by gerardforcada** [deployphp/deployer#3356] [General]
Hey there,

I've seen some tasks can be ignored if the .env file is missing ([reference](https://github.com/deployphp/deployer/blob/f51f0e3c03341251bdc1bd1561329678b3dbfdcd/recipe/laravel.php#L112))

I was wondering if it's a good idea to add this flag to all the database related tasks ([these ones](https://github.com/deployphp/deployer/blob/f51f0e3c03341251bdc1bd1561329678b3dbfdcd/recipe/laravel.php#L108-L121))

It doesn't make sense to me that the `php artisan migrate` command gets skipped, but then (for example) if the deployment fails and you want to roll back the database changes, it fails because there's no .env file.

If I created a PR with this change, would you accept it? Or is there any reason why those commands wouldn't need to be skipped?

**Comment by peterjaap** [comment]
Good idea! You're right, that makes no sense. PR it up and I'll accept it :)

**Reply by gerardforcada** [reply]
@peterjaap Cool, thank you very much! I have created the [PR](https://github.com/deployphp/deployer/pull/3359) 🎉

**Reply by peterjaap** [reply]
Great!
